### ISSUE TITLE
`valid-jsdoc` not erroring as documented

### ISSUE THREAD

**Issue by IanVS** [eslint/eslint#3674]
According to the documentation, all of the following should give errors, but they do not:

```js
/*eslint valid-jsdoc: 2*/

// missing type for @param and missing @returns
/**
 * A description
 * @param num1 The first number.
 */

// missing description for @param
/**
 * A description
 * @param {int} num1
 */

// no description for @returns
/**
 * A description
 * @returns {int}
 */

// no type for @returns
/**
 * A description
 * @returns Something awesome
 */
```

**Comment by IanVS** [comment]
Nevermind, an error is not reported unless there is actually a function under the comment.  When inserted, errors occur as expected.
